_layouts/index.html

```diff
@@ -54,74 +54,88 @@
       <div id="informations">
         <ul>
           <li>
+            <h2>How Do I Install Homebrew?</h2>
```

Comment:
Yeah, new strings are difficult on a website that has already been translated so widely. I tried to keep the two strings fairly accessible in terms of word choice but I appreciate native language support is a better solution.